src/components/settings/SettingsUiSettingsTab.vue

```diff
@@ -341,7 +341,7 @@ export default class SettingsUiSettingsTab extends Mixins(BaseMixin) {
     }
 
     get disableFanAnimation() {
-        return this.$store.state.gui.uiSettings.hideSaveConfigForBedMash ?? false
+        return this.$store.state.gui.uiSettings.disableFanAnimation ?? false
```

Comment:
please create a separate PR for this bugfix

Reply:
done: https://github.com/mainsail-crew/mainsail/pull/1381